dataplane/apigen/ccgen/servergen.go

```diff
@@ -179,6 +179,26 @@ class {{ .ServiceName }} final : public lemming::dataplane::sai::{{ .ServiceName
 {{ range .Funcs }}
 {{- if and .ProtoRPCName (not .IsStreaming) }}
 grpc::Status {{ $.ServiceName }}::{{ .ProtoRPCName }}(grpc::ServerContext* context, const lemming::dataplane::sai::{{ .ProtoRequestType }}* req, lemming::dataplane::sai::{{ .ProtoResponseType }}* resp) {
+	LOG(INFO) << "Func: " << __PRETTY_FUNCTION__;
+	{{- if .UseCommonAPI }}
+	{{ if eq .Operation "remove" }}
+	grpc::ClientContext context;
+	{{ if .OidVar -}} auto status = api->{{ .Name }}(req.get_oid()); {{ end }}
```

Comment:
req is pointer, should be req->get_oid()

Reply:
Done